src/js/components/Peak/PeakDescription.tsx

```diff
@@ -8,8 +8,14 @@ import PeakStatistics from './PeakStatistics';
 export type PeakDescriptionProps = {
   peak: Peak;
   showForm: boolean;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  statistics: any;
```

Comment:
A czemu any?

Reply:
Bo zwracany jest dosyć skomplikowany obiekt i stwierdziłam, że prościej będzie dać any niż robić pod niego model

Reply:
Chyba, że jest coś lepszego niż any co nie wymaga robienia modelu?

Reply:
możesz zadeklarować że jest to obiekt 
```
  export type statistics = {
    [key: string] : number | string | coś tam
}
```

Reply:
Ale i tak lepiej będzie skoro to statyczna wartość to zrobić po prostu console.log obiektu i napisać model

Reply:
Ok, wrzuciłam PR z modelem